### PR TITLE
add Contains Function

### PR DESCRIPTION
**Pull request by devcer**

### CODE REVIEW

**Comment by devcer** [comment]
Reference: https://github.com/jreina/ShittyLINQ.js/issues/6

File: src/Contains.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/**
+ * @template T
+ * @param {T} element
+ * @returns {boolean}
+ */
+function Contains(element) {
+  return this.indexOf(element) === -1 ? false : true;
```
Comment (jreina) [inline src/Contains.js]: Does this need to be a ternary?

Comment (devcer) [reply src/Contains.js]: The more type checking the better it would be.

Comment (jreina) [reply src/Contains.js]: The ternary is not needed. Please change to `this.indexOf(element) !== -1`

**Comment by devcer** [comment]
Hi, Made the required changes by removing the ternary operators. I think my editor has fixed some linting errors in other files. hope it is fine. else let me know so that i can remove the other files from my commit.

**Comment by jreina** [comment]
Looks good to me! Thank you for your contribution. Happy hacking 🎃

**Pull request merged**